README.md

```diff
@@ -124,7 +124,7 @@ And this is an example citation format:
 
 # License
 
-[![license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/poliastro/poliastro/raw/main/COPYING)
+[![license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://raw.githubusercontent.com/poliastro/poliastro/main/COPYING)
```

Comment:
```suggestion
[![license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/poliastro/poliastro/raw/main/COPYING)
```
This can be removed in my opinion as it was fixed with your previous PR :)